### DISCUSSION TITLE
How to make radar chart using redash?

### DISCUSSION THREAD

**Discussion by qq20230530** [getredash/redash#6094] [Q&A]
Has anyone ever used redash to create radar chart?
I need radar charts on the dashboard , but do not know where to start?
Thank you.

**Comment by gaecoli** [comment]
> Has anyone ever used redash to create radar chart? I need radar charts on the dashboard , but do not know where to start? Thank you.

There is no radar chart.

**Reply by qq20230530** [reply]
I have noticed that redash does not provide radar chart option.
But aren't there any other methods to create radar chart on dashboard? Like customizing.
